vizro-core/mkdocs.yml

```diff
@@ -43,6 +43,7 @@ nav:
       - Models: pages/API-reference/models.md
       - Data manager: pages/API-reference/manager.md
       - Actions: pages/API-reference/actions.md
+      - CapturedCallables: pages/API-reference/captured-callables.md
```

Comment:
Is this actually relevant for anything other than tables? Should we call it "Table functions" or something instead?

Reply:
Ok, let's call it that for now, and then later generalise when we have charts etc there 👍